packages/component-boilerplate/package.json

```
@@ -35,26 +40,19 @@
     "typescript": "^4.1.5"
   },
   "dependencies": {
+    "@formatjs/intl": "^1.8.2",
     "@saasquatch/dom-context-hooks": "^1.0.0",
     "@saasquatch/universal-hooks": "^1.0.0",
     "@wry/equality": "^0.3.3",
     "debounce": "^1.2.0",
-    "dom-context": "^1.0.1",
+    "dom-context": "^1.2.0",
     "fast-memoize": "^2.5.2",
     "graphql": "^15.5.0",
+    "graphql-combine-query": "^1.2.3",
     "graphql-request": "^3.4.0",
     "history": "^5.0.0",
-    "jsonpointer": "^4.1.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/saasquatch/program-tools.git",
-    "directory": "packages/component-boilerplate"
-  },
-  "author": "SaaSquatch Team",
-  "bugs": {
-    "url": "https://github.com/saasquatch/program-tools"
-  },
-  "homepage": "https://github.com/saasquatch/program-tools",
-  "license": "ISC"
+    "jsonpointer": "^4.1.0",
+    "rxjs": "^6.6.7",
+    "uuid": "^8.3.2"
```

Should use `nanoid` instead of `uuid`

```suggestion
    "uuid": "^8.3.2"
```

---

👍